openverse_catalog/dags/providers/provider_api_scripts/inaturalist.py

```diff
@@ -135,31 +243,125 @@ def create_preingestion_tasks():
                         f"{file_name}.csv.gz" for file_name in SOURCE_FILE_NAMES
                     ],
                 },
+                doc_md="Has iNaturalist published new data to s3?",
```

Comment:
Do you mind rephrasing this in a more declarative form? 🙂

Reply:
Done.